wagtail/admin/urls/__init__.py

```diff
@@ -82,9 +86,23 @@
 # Add "wagtailadmin.access_admin" permission check
 urlpatterns = decorate_urlpatterns(urlpatterns, require_admin_access)
 
+sprite_hash = None
+
+
+def get_sprite_hash():
+    global sprite_hash
+    if not sprite_hash:
+        sprite = render_to_string("wagtailadmin/shared/icons.html", icons())
+        sprite_hash = hashlib.sha1(
+            (sprite + settings.SECRET_KEY).encode("utf-8")
```

Comment:
What's the purpose of SECRET_KEY here?

Reply:
I did the same as in https://github.com/wagtail/wagtail/blob/33e4c0ec7b889131740ee66a954f4ca2293fc2dd/wagtail/admin/staticfiles.py#L31-L36

I assume this is to keep the Wagtail version a secret.